docs/release-notes/8.4.0-2.md

```diff
@@ -0,0 +1,92 @@
+#  Percona XtraBackup 8.4.0-2 (2024-12-16)
+
+Get started with [Quickstart Guide for Percona XtraBackup].
+
+## Release highlights
+
+Percona XtraBackup 8.4.0-2 merges the MySQL 8.4 code base and introduces Percona XtraBackup 8.4.0-2 Pro build, in addition to the regular community builds.
+
+Percona XtraBackup 8.4.0-2 Pro build includes the [capabilities](pxb-pro.md#capabilities) that are typically requested by large enterprises. These Pro builds are specifically created and tested by Percona, and the corresponding packages are supported only for Percona customers with a valid subscription. 
+
+Percona XtraBackup 8.4.0-2 Pro build is available for the following platforms:
+
+* Oracle Linux 9
+
+* Ubuntu (22.04)
+
+* Ubuntu (24.04)
+
+* Debian (12)
+
+Percona XtraBackup 8.4.0-2 Pro build includes the [`Reduced backup lock time`](../reduction-in-locks.md) feature. With the addition of the [`--lock-ddl=REDUCED`](./xtrabackup-option-reference.md#lock-ddl) option this feature minimizes the time the server remains locked by xtrabackup during both full and incremental backups. Now, you can execute `Data Definition Language` (DDL) operations while the backup is in progress.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option features are as follows:
+
+* Acquire and release the backup lock quickly: The backup process minimizes the time it holds the backup lock, allowing for concurrent DDL operations.
+
+    ??? note "Comparing the backup duration with --lock-ddl=REDUCED"
+
+        While we strive to provide accurate performance benchmarks, real-world results may vary depending on your hardware and software configurations.
+
+        The following tables illustrate the approximate time differences between `--lock-ddl=ON` and `--lock-ddl=REDUCED` for local and cloud backups (Amazon S3). The "Improvement (X times)" column shows how many times faster the backup is when using `--lock-ddl=REDUCED` compared to `--lock-ddl=ON`.
```

Comment:
column shows how much less the Server is locked by xtrabackup using..

Reply:
updated